cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go

```diff
@@ -603,10 +603,8 @@ func TestValidateFeatureGates(t *testing.T) {
 		featureGates featureFlag
 		expected     bool
 	}{
-		{featureFlag{"SelfHosting": true}, true},
-		{featureFlag{"SelfHosting": false}, true},
-		{featureFlag{"StoreCertsInSecrets": true}, true},
-		{featureFlag{"StoreCertsInSecrets": false}, true},
+		{featureFlag{"Unknown": true}, false},
+		{featureFlag{"Unknown": false}, false},
 		{featureFlag{"Foo": true}, false},
```

Comment:
i think Foo is the same as Unknown here.
please remove the line with Foo.

also please add these valid feature gate tests:
```
{featureFlag{"CoreDNS": true}, true},
{featureFlag{"CoreDNS": false}, true},
```

Reply:
:+1: